src/xcode/ENA/ENA/Source/Scenes/Settings/BackgroundAppRefresh/BackgroundAppRefreshViewModel.swift

```diff
@@ -0,0 +1,205 @@
+
+class BackgroundAppRefreshViewModel {
+
+	enum CombinedBackgroundAppRefreshStatus {
+		case on
+		case off
+		case offInPowerSaving
+	}
+	
+	// MARK: - Init
+
+	init(
+		backgroundRefreshStatusProvider: BackgroundRefreshStatusProviding = UIApplication.shared,
+		lowPowerModeStatusProvider: LowPowerModeStatusProviding = ProcessInfo.processInfo,
+		onOpenSettings: @escaping () -> Void,
+		onShare: @escaping () -> Void
+	) {
+		self.backgroundRefreshStatusProvider = backgroundRefreshStatusProvider
+		self.lowPowerModeStatusProvider = lowPowerModeStatusProvider
+		self.onOpenSettings = onOpenSettings
+		self.onShare = onShare
+
+		backgroundRefreshStatus = backgroundRefreshStatusProvider.backgroundRefreshStatus
+		lowPowerModeEnabled = lowPowerModeStatusProvider.isLowPowerModeEnabled
+		observeBackgroundAppRefresh()
+		observeLowPowerMode()
+		calculateCombinedBackgroundStatus()
+	}
+	
+	// MARK: - Internal
+
+	let title = AppStrings.BackgroundAppRefreshSettings.title
+	let subTitle = AppStrings.BackgroundAppRefreshSettings.subtitle
+	let description = AppStrings.BackgroundAppRefreshSettings.description
+	
+	let settingsHeaderTitle = AppStrings.BackgroundAppRefreshSettings.Status.header.uppercased()
+	let backgroundAppRefreshTitle = AppStrings.BackgroundAppRefreshSettings.Status.title
+
+	@Published var backgroundAppRefreshStatusText: String = ""
+	@Published var backgroundAppRefreshStatusAccessibilityLabel: String = ""
+	@Published var image: UIImage?
+	@Published var showInfoBox: Bool = false
+	@Published var infoBoxViewModel: InfoBoxViewModel?
+
+
+	// MARK: - Private
+
+	private let backgroundRefreshStatusProvider: BackgroundRefreshStatusProviding
+	private let lowPowerModeStatusProvider: LowPowerModeStatusProviding
+	private let onOpenSettings: () -> Void
+	private let onShare: () -> Void
+
+    private var subscriptions = Set<AnyCancellable>()
+
+	private let infoBoxTitle = AppStrings.BackgroundAppRefreshSettings.InfoBox.title
+	private let infoBoxDescriptionOff = AppStrings.BackgroundAppRefreshSettings.InfoBox.description
+	private let infoBoxDescriptionLowPowerMode = AppStrings.BackgroundAppRefreshSettings.InfoBox.lowPowerModeDescription
+	private let infoBoxTitleImage = UIImage(named: "Icons_iOS_Hintergrundaktualisierung_Aus")
+	private let buttonTextShare = AppStrings.BackgroundAppRefreshSettings.shareButtonTitle
+	private let buttonTextSettings = AppStrings.BackgroundAppRefreshSettings.openSettingsButtonTitle
+
+	private var backgroundRefreshStatus: UIBackgroundRefreshStatus {
+		didSet {
+			calculateCombinedBackgroundStatus()
+		}
+	}
+
+	private var lowPowerModeEnabled: Bool {
+		didSet {
+			calculateCombinedBackgroundStatus()
+		}
+	}
+
+	private var combinedBackgroundAppRefreshStatus: CombinedBackgroundAppRefreshStatus = .off {
+		didSet {
+			switch combinedBackgroundAppRefreshStatus {
+			case .on:
+				backgroundAppRefreshStatusText = AppStrings.BackgroundAppRefreshSettings.Status.on
```

Comment:
Do we need here `showInfoBox = false`?

Reply:
Nice finding, this was an old property which is no longer used. 
I removed `@Published var showInfoBox: Bool = false`